Practice 2/simple_xor.c

```diff
@@ -0,0 +1,74 @@
+#include <stdio.h>
+
+int converter(int dec) {
```

Comment:
Please don't mix multiple home tasks in one pull request - it must be separate pull request for each home task